DigitalSignageUAP/SlideshowPage.xaml.cs

```diff
@@ -307,8 +307,20 @@ void PlayAudio()
                             DisplayObject DO = new DisplayObject();
                             string filename = fileElement.Attribute("path").Value.Substring(fileElement.Attribute("path").Value.LastIndexOf('/') + 1);
```

Comment:
The local paths use backslashes ("\") while the URLs use forward slashes ("/"). Therefore LastIndexOf('/') above will only affect the URL paths. Filenames in subfolders on the local filesystem should be resolved correctly as defined in the Assets\config.xml file.